### DISCUSSION TITLE
(macOS) Fail to build

### DISCUSSION THREAD

**Discussion by astudnev** [filecoin-project/lotus#8223] [Build and Sync]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [X] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
1.14.3
```

### Describe the Bug

make all fails with the error

```
go version
go version go1.16.5 darwin/amd64
studnev@MacBook-Pro-Aleksej lotus % make all  
/Library/Developer/CommandLineTools/usr/bin/make -C extern/filecoin-ffi/ .install-filcrypto
go clean -cache -testcache .
go: github.com/filecoin-project/specs-actors/v7@v7.0.0-20211117170924-fd07a4c7dff9: invalid version: unknown revision fd07a4c7dff9
make[1]: *** [.install-filcrypto] Error 1
make: *** [build/.filecoin-install] Error 2
```

### Logging Information

```text
go: github.com/filecoin-project/specs-actors/v7@v7.0.0-20211117170924-fd07a4c7dff9: invalid version: unknown revision fd07a4c7dff9
make[1]: *** [.install-filcrypto] Error 1
make: *** [build/.filecoin-install] Error 2
```

### Repo Steps

git checkout 
make all

**Comment by Reiers** [comment]
Hi @astudnev 

Sending you to Discussions for help and troubleshooting.

Thanks for the ticket!

**Comment by LexLuthr** [comment]
@astudnev Can you try to pull all the dependencies before hand and make sure go proxy is not set. I have seen similar issue due to go proxy.

git submodule update --init

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new update from the author for more than 7 days.
